services/gundeck/test/unit/MockGundeck.hs

```diff
@@ -253,8 +262,8 @@ genClientId = newClientId <$> arbitrary
 
 genProtoAddress :: Gen (UserId -> ClientId -> Address "no-keys")
 genProtoAddress = do
-  _addrTransport <- QC.elements [minBound..]
-  arnEpId <- arbitrary
+  _addrTransport :: Transport <- QC.elements [minBound..maxBound]
```

Comment:
why the `maxBound`?

Reply:
I thought it clarified the intent better (`[minBound..]` looks line an infinite list), but I grepped now and we have `[minBound..]` in other places as well so I'll revert this change.

Reply:
ok, fair enough.  i have no strong opinion then, feel free to leave it in  :-)

(i like to leave it out because if it's an error, it will always trigger on the first run that touches this, and it's prettier because fewer bytes of source code.  more zen.)